ci/benchmark/build.sh

```diff
@@ -63,6 +63,11 @@ nvidia-smi
 logger "Activate conda env..."
 source activate rapids
 
+
+# Enter dependencies to be shown in ASV tooltips.
+CUGRAPH_DEPS=(cudf rmm)
+LIBCUGRAPH_DEPS=(cudf rmm)
```

Comment:
You could drop the `LIBCUGRAPH_DEPS` here since we don't have benchmarks for that, and just collapse all deps into `CUGRAPH_DEPS`. Since the deps are the same for now, you'd end up just deleting `LIBCUGRAPH_DEPS`.

I'm assuming you have both as more of a placeholder for eventual C++ benches?  If so, I usually like that idea but cugraph may never have those (other repos may though), and if you choose to remove `LIBCUGRAPH_DEPS` you can probably remove some of the other code (that removes duplicates, etc.) too.  It's up to you but I'm leaning more towards removing and saving this code somewhere for use on repos that have multiple benchmark types.

Reply:
i'm going to leave this for now. there's some code that appears later on in the script that depends on `LIBCUGRAPH_DEPS` and I don't want to remove that also since it will serve as a good template for copying this script to other repos that do have python and C++ benchmarks.